uproot/interp/auto.py

```diff
@@ -192,6 +194,14 @@ def transform(node, tofloat=True):
             if len(branch.fBranches) > 0 and all(len(x.fLeaves) == 1 and x.fLeaves[0].fLeafCount is branch.fLeaves[0] for x in branch.fBranches):
                 return asdtype(">i4")
 
+            if isinstance(branch._streamer, uproot.rootio.TStreamerObject):
+                for x in all_classes:
+                    if (branch._streamer.fTypeName == x._classname):
+                        try:
+                            return asobj(x, branch._context)
+                        except:
```

Comment:
This `try-catch` is too wide— hiding too many exceptions will make it difficult to debug later. What exception exactly are you trying to catch? Under what conditions does it happen? Can you do `if` statements for that instead, so that surprising exceptions still get raised?

If you're copying something I did, please point out my bad example.